src/components/prices/PriceFeed.tsx

```diff
@@ -0,0 +1,70 @@
+import { prices } from '@defichain/whale-api-client'
+import Image from 'next/image'
+import { IoAlertCircleOutline, IoCheckmarkCircleOutline } from 'react-icons/io5'
+import { MdShowChart } from 'react-icons/md'
+import NumberFormat from 'react-number-format'
+import { getPriceCopy, PriceCopy } from '../../cms/prices'
+
```

Comment:
I know this is something trivial but i think it will  be better if import are arranged alphabetically for clarity